pkg/gui/gui.go

```diff
@@ -136,6 +142,17 @@ func (gui *Gui) scrollDownMain(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) scrollDownCommandStatus(g *gocui.Gui, v *gocui.View) error {
```

Comment:
this duplicates some code from the above scroll method. Could we make a generic function that takes a view name and returns a method for scrolling that view? Something like this:

```
func (gui *Gui) genericScrollDown(viewName string) func(g *gocui.Gui, v *gocui.View) error {
	return func (g *gocui.Gui, v *gocui.View) error {
		view, err := g.View(viewName)
		if err != nil {
			return err
		}
		ox, oy := view.Origin()
		if oy < len(view.BufferLines()) {
			return view.SetOrigin(ox, oy+gui.Config.GetUserConfig().GetInt("gui.scrollHeight"))
		}
		return nil
	}	
}
```
then in our keybindings file we can use it like so:
```
		}, {
			ViewName: "",
			Key:      gocui.KeyPgdn,
			Modifier: gocui.ModNone,
			Handler:  gui.genericScrollDown("main"),
		}, {
```